.storybook/themes/commerce.scss

```diff
@@ -632,12 +632,12 @@
 					)
 				),
 				"hero-text-overlay": (
-					"margin": 0auto,
+					"margin": 0 auto,
```

Comment:
These values are incorrect in the `commerce.json` files of the blocks as well. I wouldn't worry about them for this ticket, but I do think we'll need one to convert the commerce only blocks to use logical properties.

Reply:
Thanks Vito, reverted all commerce changes